applicationconfig/client/src/main/java/com/azure/applicationconfig/ConfigurationClient.java

```diff
@@ -2,39 +2,35 @@
 // Licensed under the MIT License.
 package com.azure.applicationconfig;
 
-import com.azure.applicationconfig.implementation.PageImpl;
+import com.azure.applicationconfig.implementation.Page;
 import com.azure.applicationconfig.implementation.RestPagedResponseImpl;
 import com.azure.applicationconfig.models.ConfigurationSetting;
-import com.azure.applicationconfig.models.ETagFilter;
-import com.azure.applicationconfig.models.Key;
-import com.azure.applicationconfig.models.KeyLabelFilter;
 import com.azure.applicationconfig.models.KeyValueCreateUpdateParameters;
-import com.azure.applicationconfig.models.KeyValueFilter;
 import com.azure.applicationconfig.models.KeyValueListFilter;
-import com.azure.applicationconfig.models.Label;
 import com.azure.applicationconfig.models.RevisionFilter;
-import com.microsoft.rest.v3.ServiceClient;
-import com.microsoft.rest.v3.http.HttpPipeline;
-import com.microsoft.rest.v3.http.policy.HttpPipelinePolicy;
-import com.microsoft.rest.v3.http.policy.RetryPolicy;
-import com.microsoft.rest.v3.http.policy.UserAgentPolicy;
-import com.microsoft.rest.v3.http.rest.RestPagedResponse;
-import com.microsoft.rest.v3.http.rest.RestResponse;
-import com.microsoft.rest.v3.implementation.RestProxy;
-import com.microsoft.rest.v3.implementation.Validator;
+import com.azure.common.ServiceClient;
+import com.azure.common.http.HttpPipeline;
+import com.azure.common.http.policy.HttpPipelinePolicy;
+import com.azure.common.http.policy.RetryPolicy;
+import com.azure.common.http.policy.UserAgentPolicy;
+import com.azure.common.http.rest.RestResponse;
+import com.azure.common.implementation.RestProxy;
+import com.azure.common.implementation.Validator;
+import org.reactivestreams.Publisher;
 import reactor.core.publisher.Flux;
 import reactor.core.publisher.Mono;
 import reactor.util.annotation.NonNull;
 
 import java.net.URL;
 import java.util.ArrayList;
 import java.util.List;
-import java.util.function.Function;
 
 /**
  * Client that contains all the operations for KeyValues in Azure Configuration Store.
  */
 public final class ConfigurationClient extends ServiceClient {
+    private static final String ETAG_ANY = "*";
```

Comment:
Is ETag a concept that is managed in the Java Pipeline? Not sure `ETAG_ANY` is exclusively for the AzConfig service...

Reply:
So far Etag has only showed up here. Can probably be moved to a common class.